Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM golang:1.23-alpine3.20 AS builder
+
+ENV GO111MODULE=on
+
+WORKDIR /gemini
+
+COPY . .
+
+RUN apk add build-base musl-dev \
+    && go build -ldflags="-s -w" -o build/gemini ./cmd/gemini
+
+FROM alpine:3.20 AS production
+
+WORKDIR /gemini
+
+RUN apk update && apk add bash musl
+
+COPY --from=builder /gemini/build/gemini .
+
+ENV PATH="/gemini:${PATH}"
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
```

Comment:
why ?
```
ENTRYPOINT ["/gemini"]
```
Is going to be enough, let's keep image as slim as possible.
We can have debug image, if you want, but generic image have to have no shell.

Reply:
Done